### ISSUE TITLE
Atom/eslint complains in every non "jasmine" file 

### ISSUE THREAD

**Issue by damianobarbati** [tlvince/eslint-plugin-jasmine#187]
**Do you want to request a *feature* or report a *bug*?**
Probably a bug because unintended behaviour.

**What is the current behavior?**
Once eslint-jasmine-plugin has been setup, Atom is complaining atop of every file (errors in the screenshot)

**If the current behavior is a bug, please provide the steps to reproduce.**
atom + the following .eslintrc:
```
{
    "parser": "babel-eslint",
    "parserOptions": {
        "ecmaVersion": 2020,
        "sourceType": "module"
    },
    "env": {
        "node": true,
        "es6": true,
        "jest": true,
        "jasmine": true
    },
    "extends": [
        "eslint:recommended",
        "plugin:jasmine/recommended",
    ],
    "rules": {
        "no-console": "off"
    }
}
```

**What is the expected behavior?**
jasmine plugin should not apply on file not ending with *.test.js or *.spec.js.

**Please mention your node.js, eslint-plugin-jasmine and operating system version.**
```
node v12.10.0
```

```
    "devDependencies": {
        "babel-eslint": "^10.0.3",
        "eslint": "^6.4.0",
        "eslint-plugin-jasmine": "^2.10.1",
        "jasmine": "^3.5.0",
        "nodemon": "^1.19.2"
    }
```

Screen of what's happening here!

<img width="997" alt="Screenshot 2019-09-28 at 19 23 17" src="https://user-images.githubusercontent.com/2225869/65820109-7ff5e700-e225-11e9-99ce-18c09fe6d8fa.png">

**Comment by damianobarbati** [comment]
Never mind, missed "plugins" section in my eslintrc.
